src/LoadingOverlay.js

```diff
@@ -85,7 +86,8 @@ LoadingOverlayWrapper.defaultProps = {
   color: '#FFF',
   zIndex: 800,
   animate: false,
-  style: {}
+  style: {},
+  onClick: null
```

Comment:
Don't need this default prop, a the default prop for the `LoadingOverlay` can handle it instead.